frontend/web/components/modals/CreateFlag.js

```diff
@@ -926,6 +984,7 @@ const CreateFlag = class extends Component {
 
               const onCreateFeature = () => {
                 this.save(createFlag, isSaving)
+                // this.save(createFlag, isSaving, metadataAdded)
```

Comment:
I think this was supposed to save the metadata

Reply:
was unnecesary, removed